assets/properties-panel.css

```
@@ -355,6 +356,10 @@
   background-color: var(--dot-color);
 }
 
+.bio-properties-panel-dot--error {
+  background-color: var(--dot-color-error);
```

BTW I just thought that if we wanted to make it even more resilient, we could re-assign the `dot-color` variable instead of changing an implementation detail of `background-color`:

```suggestion
  --dot-color: var(--dot-color-error);
```

Still, it's good enough as is.